.github/workflows/integration-tests.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         operating-system:
           - ubuntu-latest
           - windows-latest
+          - macos-latest
```

Comment:
This matrix value is unused because in `runs-on` there's still ubuntu-latest.

But I don't think there's much value in this. Rather I'd like to have a separate workflow for Windows-only specific issues.

Reply:
But that is impossible to say upfront ;-)